src/index.html

```diff
@@ -5,9 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="iMac A1419">
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__review-container">
+
+        <div class="stars stars--4">
+
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+
+        </div>
+
+        <div class="card__review-count">Reviews: 5</div>
+      </div>
+
+      <div class="card__price-container">
+
+        <div class="card__price-label">Price:</div>
+
+        <div class="card__price">$2,199</div>
+
+      </div>
+
+      <a
+        class="card__buy"
+        href="#"
+        data-qa="hover"
+        >
+        Buy
+      </a>
```

Comment:
it should be `button` I guess, not link

Reply:
Tips & Hints
Add ALL data-qa attributes required in the task
If you use SCSS, check background-image: url() to be relative to the main.scss. So should start with ../images.
Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.
There should be only 2 BEM blocks in your code card and stars all the others are just card elements
Item with the text Buy should be a link.  ←----- I DID RIGHT!!!
Check font styles on the mockup. Use [google fonts](https://fonts.google.com/)